test/singledc/pb_client_SUITE.erl

```diff
@@ -116,7 +116,7 @@ get_empty_crdt_test(_Config) ->
     Bound_object = {<<"key1">>, antidote_crdt_counter_pn, Bucket},
 
     {ok, Pid} = antidotec_pb_socket:start(?ADDRESS, ?PORT),
-    {ok, TxId} = antidotec_pb:start_transaction(Pid, ignore, []),
+    {ok, TxId} = antidotec_pb:start_transaction(Pid, term_to_binary(ignore), []),
```

Comment:
Would probably be cleaner to call `term_to_binary` in the antidote client code.

Reply:
done